documentation/src/docs/controller-service.md

```diff
@@ -128,7 +128,11 @@ have two policies that users can define, namely [**Scaling** **Policy**](https:/
        - **Scaling policy** describes if and under what circumstances a Stream should automatically scale its number of Segments.  
        - **Retention policy** describes a policy about how much data to retain within a Stream based on **time** (*Time-based Retention*) and data **size** (*Size-based Retention*).
 
-  3. [**Transaction**](pravega-concepts.md#transactions) **Management**: Implementing Transactions requires the manipulation of Stream Segments. With
+  3. **Tag Management**: The Controller is responsible for storing user-defined Tags for a given Stream.
+     Tags are singleton labels that are useful to specify identifying attributes of a Stream, meaningful and relevant to users.
+     The Controller indexes these Tags and provides APIs by which the user  can organize and select a subset of Streams under a given Scope.
```

Comment:
perhaps we need to say organize and select a subset of streams under a scope that share the same tag.

Reply:
done.